src/app/components/select-items/select-items.component.spec.ts

```diff
@@ -13,11 +14,38 @@ describe('SelectItemsComponent', () => {
       .compileComponents();
 
     fixture = TestBed.createComponent(SelectItemsComponent);
+    fixture.componentRef.setInput('possibleItemNames', possibleItemNames);
     component = fixture.componentInstance;
     fixture.detectChanges();
   });
 
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  it('should add empty item', () => {
+    component.addItem();
+    expect(component.getSelectedItem()).toBe('');
+  });
+
+  it('should display a list of all possible items', () => {
+    const items = fixture.nativeElement.querySelectorAll('option');
+    const numberOfDefaultItems = 1
+    expect(items.length).toBe(possibleItemNames.length + numberOfDefaultItems);
+  })
```

Comment:
Here instead of checking the length of the items, we should test the actual content of the items rendered.
So that if any fruit is removed or added to the options list, our tests fail and we know what to do to fix them.